pkgs/development/tools/vim-vint/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, pkgs, python3Packages, stdenv }:
+
+python3Packages.buildPythonApplication rec {
+  name = "vim-vint-${version}";
+  version = "0.3.11";
+
+  src = pkgs.fetchurl{
+    url = "mirror://pypi/v/vim-vint/${name}.tar.gz";
+    sha256 = "cda7090116b3aea12bdfe8ed44e71ead739df4e7e0ad24e27bb87754dedb613b";
+  };
+
+  # OSX keeps getting error: `Download error on https://pypi.python.org/simple/coverage: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:720) -- Some packages may not be found!`
+  # To amend this, we use nix for the packages (instead of letting setup.py fetch them).
+  # Vim-vint's test-requirements.txt is hardcoded with specific package versions, which results in errors, so we patch it
+  patchPhase = stdenv.lib.optionalString stdenv.isDarwin ''
+    substituteInPlace setup.py --replace "load_requires_from_file('test-requirements.txt')" "[]"
+    '';
+  buildInputs = with python3Packages; stdenv.lib.optionals stdenv.isDarwin [ coverage pytest pytestcov ];
```

Comment:
An even better solution: realize that the pypi version of this pacakge (vim-vint) doesn't ship with tests, so I can just skip the buildinputs anyway, for as long as I do the patchPhase. Waiting for travis to verify that it still works...

I'm not sure, is it better to get version 0.3.11 from [github](https://github.com/Kuniwak/vint/releases), where they ship version 0.3.11 with test?